azure-pipelines.yml

```diff
@@ -0,0 +1,21 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger:
+- master
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '10.x'
```

Comment:
It might be good to pin this to a specific version, so that our personal machines can match whats used in CI.

Reply:
Is this the node version?

Reply:
Sorry I didn't mean to create this PR. Although it would be good to agree on a node version.